easybuild/easyconfigs/l/LMfit/LMfit-0.9.14-intel-2018b-Python-2.7.15.eb

```
@@ -0,0 +1,41 @@
+easyblock = 'PythonBundle'
+
+name = 'LMfit'
+version = '0.9.14'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://lmfit.github.io/lmfit-py'
+description = "Lmfit provides a high-level interface to non-linear optimization and curve fitting problems for Python"
+
+toolchain = {'name': 'intel', 'version': '2018b'}
+
+dependencies = [('Python', '2.7.15')]
+
+use_pip = True
+sanity_pip_check = True
+
```

Since all `source_urls` are from pypi you can add the following line here and remove all following declarations for `source_urls`
```suggestion
exts_default_options = {'source_urls': [PYPI_SOURCE]}
```